### PR TITLE
Initial version of PagerDuty Notification documentation

### PR DESCRIPTION
**Pull request by waab76**
This change adds documentation for PagerDuty Event Notifications.

Screenshot of the text:
Screen Shot 2020-10-28 at 3 08 21 PM

The inline image added:
Screen Shot 2020-10-28 at 12 42 42 PM

### CODE REVIEW

File: pages/alerts.rst [hunk 1]
```diff
@@ -240,6 +240,50 @@ Here is an example of the payload included in a notification::
...
+.. note:: PagerDuty alert notifications are intended as a replacement for the PagerDuty
+   integration previously available from 
+   `Graylog Labs <https://github.com/graylog-labs/graylog-plugin-pagerduty>`__.  If you 
+   are using the Graylog Labs PagerDuty integration, you should migrate to the officially
+   supported PagerDuty alert notifications at your earliest convenience.
+
+.. important:: In order to use PagerDuty alert notifications, you must create a new 
+   integration in PagerDuty to obtain an integration routing key:     
+   `PagerDuty Documentation <https://developer.pagerduty.com/docs/events-api-v2/overview/#getting-started>`__ 
+
```
Comment (ChristopherKB) [inline pages/alerts.rst:260]: Do you need a new integration if you are already using the PagerDuty Integration? We should probably state that if so.

**Review by ChristopherKB — APPROVED** [review]
Looks good.

**Review by jalogisch — APPROVED** [review]
LGTM

File: pages/alerts.rst (hunk 1 above)
Comment (waab76) [reply pages/alerts.rst:260]: Yeah, I can clarify that you only need to create a new integration if you don't already have one.

Comment (waab76) [reply pages/alerts.rst:260]: Updated text

Screen Shot 2020-10-29 at 11 08 22 AM

**Pull request merged**